src/components/Layout/Checkbox.tsx

```diff
@@ -70,34 +80,41 @@ const Toggle: FunctionComponent<Toggle> = ({ isChecked }) => {
 
 interface Checkbox extends Toggle {
   label: ReactElement;
+  addons: ReactElement | null;
   onToggle: (isChecked: boolean) => void;
 }
 
 export const Checkbox: FunctionComponent<Checkbox> = ({
   label,
+  addons,
   isChecked,
   onToggle,
 }) => {
   return (
-    <TouchableHighlight
-      onPress={() => {
-        onToggle(!isChecked);
-        if (Platform.OS === "android") {
-          Vibration.vibrate(10);
-        }
-      }}
-      underlayColor="transparent"
-      activeOpacity={1}
+    <View
```

Comment:
Given that we're stopping the propagation of the event, can we keep this as a TouchableHIghlight?